src/speakbuttons.h

```diff
@@ -79,6 +82,7 @@ private slots:
     QMediaPlayer *m_mediaPlayer = nullptr;
     QOnlineTts::Voice m_yandexVoice = QOnlineTts::NoVoice;
     QOnlineTts::Emotion m_yandexEmotion = QOnlineTts::NoEmotion;
+    QMap<QOnlineTranslator::Language, QLocale::Country> m_googleRegions;
```

Comment:
I'm not sure if using this class as temporary storage for regions is a good idea.
What if instead we store it inside the combobox using [itemData](https://doc.qt.io/qt-5/qcombobox.html#itemData)?

Reply:
Sorry, I'm not sure I understand. As far as I know, there are two `SpeakButton`s in the main window, where there's no combo box to help store its state. So how can the two speakers remember region settings without this member?

Reply:
Oh, sorry, I thought it used only in settings. Also looks like I already store voice and emotion settings.
I don't like my decision of storing this data in this class :smile: But for your PR keep it as is for consistency. You did everything right.